packages/tic-tac-toe/config/env.js

```diff
@@ -32,7 +32,7 @@ const REACT_APP = /^REACT_APP_/i;
 // These are Magmo specific values that should be injected into the application
 const MAGMO_VALUES = [
```

Comment:
This seems like an out-of-date name

Reply:
I copy pasted this from somewhere and I don't remember where :)
I don't even think I need this file anymore hmm. :)